exercises/F-strings-methods/exercise.js

```diff
@@ -1,3 +1,6 @@
 // Start by creating a variable `message`
+let firstname = "shima";
+let nameLength = firstname.length;
+console.log(nameLength);
 
-console.log(message);
+ console.log ("My name is " + firstname +  " and it has " +  nameLength + " characters");
```

Comment:
This is perfect, the only comment again would be let to const, but this feedback applies throughout the entire PR when the value isn't changed so I'll stop repeating it.

You did a great job here Shima :)